partner_title_active/README.rst

```diff
@@ -0,0 +1,78 @@
+====================
+Partner title active
```

Comment:
That file should be split for unit files (like `DESCRIPTION.md` or `CONTRIBUTORS.md`), shouldn't?

Reply:
yep, but this should be generated by the pre-commit
here it wasn't the case, probably it will be after merge